base/gfspclient/downloader.go

```diff
@@ -36,6 +36,31 @@ func (s *GfSpClient) GetObject(
 	return resp.GetData(), nil
 }
 
+func (s *GfSpClient) GetPiece(
+	ctx context.Context,
```

Comment:
maybe move these parameters to one line.

Reply:
Updated.